### ISSUE TITLE
Delphi 10.3 cannot be used

### ISSUE THREAD

**Issue by gtyhn** [neslib/DelphiLearnOpenGL#2]
10.3 Cannot be used on Android

**Comment by neslib** [comment]
Yes, Embarcadero changed to application model on Android (again).
I don't have time to look into this soon.
Since this is "just" a learning repository, you can stick with the non-Android platforms for now, or use Delphi 10.2 for Android.

Of course, a pull request with fixes is also appreciated ;)

**Comment by gtyhn** [comment]
By debugging on Android, I found out that because onNativeWindowCreated didn't trigger the callback, the APP_CMD_INIT_WINDOW message could not be passed, and in TPlatformAndroid.HandleAppCmd ​​it was the APP_CMD_INIT_WINDOW message to initialize the NativeWindow related environment, so I don't know why onNativeWindowCreated is not called by Android.

Sorry, my English is very poor
